docs/user/supported_devices.rst

```diff
@@ -192,7 +192,11 @@ ath79-generic
   - WiFi pro 1750i
   - WiFi pro 1750x
 
-  * OCEDO
+* GL Innovations
```

Comment:
Rename this to `GL-iNet`. I've never read the company calling themselves GL Innovations. If at all, this is long long ago.

Looks good otherwise.

Reply:
i copied this from other target, renaming there too?

Reply:
Yep, lets do it.

Reply:
done, please check

Reply:
@rotanid It's nitpicky, but shouldn't it be `GL.iNet` instead of `GL-iNet`? (As I've proposed in my first commit)

Reply:
i don't mind how it's called, you suggested to use GL-iNet, now it's GL.iNet, and mweinelt is also in the loop.
please come to an agreement about it and i will do it.

Reply:
-> `GL.iNet`

Reply:
@rotanid Sorry, I've mistyped my proposal in the initial commit. Realized it just now.

LGTM

Reply:
done, please check and maybe merge